src/components/mapexplorer.js

```diff
@@ -393,16 +392,9 @@ export default function ({
 
   const {name, lastupdatedtime} = currentHoveredRegion;
 
-  /* useEffect(() => {
-    let found = false;
-    stateTestData.forEach((testObj, index) => {
-      if (testObj.state === panelRegion.name) {
-        found = true;
-        setTestObj(testObj);
-      }
-    });
-    if (!found) setTestObj({});
-  }, [panelRegion, stateTestData, testObj]);*/
+  useEffect(() => {
+    setTestObj(stateTestData.find((obj) => obj.state === panelRegion.name));
```

Comment:
This takes the first entry in the list. We should take the last entry in the list (test data is available per state per date basis in an ascending order of dates).

Reply:
I didn't bother to look at the format. I've reversed the data when it's loaded. Hope that's fine